src/main/webapp/web/css/Inboxhub.css

```
@@ -120,7 +120,22 @@ input {
     overflow-y: auto;
-    overflow-x: hidden;
+    overflow-x: auto;
+}
+
+#inbox_table_wrapper {
+    max-width: 100%;
+}
+
+#inbox_table_wrapper > .row {
+    --bs-gutter-x: 0;
+    margin-left: 0;
```

Stylelint (17.11.1)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)